lib/src/lib_ui_components/show_modal.dart

```diff
@@ -240,10 +240,10 @@ class _ModalContent extends StatelessWidget {
       );
 
   Widget _buildCloseButton(BuildContext context) => Container(
-        padding: const EdgeInsets.symmetric(vertical: 10),
```

Comment:
Please use the theme extension, instead of hardcoding values

Reply:
This padding was added to theme extensions